content/swiftui/concepts/layouts/layouts.md

```diff
@@ -0,0 +1,166 @@
+---
+Title: 'Layouts'
+Description: 'Layout is a protocol in SwiftUI used to create layouts for user interfaces. Layouts specify how to arrange the views they contain.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'UI'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Layout`** is a `protocol` in [SwiftUI](https://www.codecademy.com/resources/docs/swiftui) used to create layouts for user interfaces. Layouts specify how to arrange the [views](https://www.codecademy.com/resources/docs/swiftui/views) they contain. It's possible to create custom and conditional layouts. There are also built-in layouts that can be used when creating conditional layouts.
```

Comment:
```suggestion
**`Layout`** is a `protocol` in SwiftUI used to create layouts for user interfaces. Layouts specify how to arrange the [views](https://www.codecademy.com/resources/docs/swiftui/views) they contain. It's possible to create custom and conditional layouts. There are also built-in layouts that can be used when creating conditional layouts.
```
We generally don't link to parent entries. Docs navigation provides those automatically.